### PR TITLE
Multi-edit fix

### PR DESCRIPTION
**Pull request by vphilippon**
@grondinjc @maym2104 The glorious PR is ready for review!

### CODE REVIEW

**Comment by maym2104** [comment]
:+1:

**Comment by vphilippon** [comment]
Wait, I'll add some more stuff

File: src/server/ideController.py [hunk 1]
```diff
@@ -257,10 +257,10 @@ def save(self):
...
                   'vers':    '<<File version>>',
                   'changes': [{
-                               'type':    '<<Type of edit (ins | del)>>',
-                               'pos':     '<<Position of edit>>',
-                               'content': '<<Content of insert | Number of deletes>>'
-                               'author':   <<If the user is the author>>
+                               'type':        '<<Type of edit (ins | del)>>',
+                               'pos':         '<<Position of edit>>',
+                               'content':     '<<Content of insert | Number of deletes>>'
+                               'is_from_you':  <<If the user is the author>>
```
Comment (grondinjc) [inline src/server/ideController.py:263]: Missing single quotes, to do in next pr

File: src/server/ideController.py [hunk 2]
```diff
@@ -425,10 +425,10 @@ def _save_callback(self, filename, changes, version, users):
...
```
Comment (grondinjc) [inline src/server/ideController.py:431]: same

**Comment by grondinjc** [comment]
Changes to do are too minor to wait.
Merging

**Pull request merged**